.expeditor/config.yml

```diff
@@ -18,23 +18,31 @@ github:
   # Delete the PR branch after successfully merged into release branch.
   delete_branch_on_merge: true
 
-# These actions are taken, in order they are specified, anytime a Pull Request is merged.
+buffers:
+  - build:
+      type: delay
+      workload_type: pull_request
+      delay: 600
```

Comment:
Should we consider something like a 1hr delay? In the typical case it's not that urgent that we get updated packages. Perhaps if there's a way to force it anyway for urgent builds? If not, I agree 10m is better.

Reply:
My vote would be start with 10 or even 5 and increase if we need to.